pyproject.toml

```diff
@@ -40,6 +40,7 @@ dev = [
     "mkdocs-material",
     "mkdocstrings[python]",
     "mypy",
+    "nox",
```

Comment:
I thought my `nox` things were already added. this should be a different PR

Reply:
Are you gonna separate them or should we just move forward with this single PR?